doc/source/api/core/typing.rst

```diff
@@ -81,43 +81,3 @@ pyvista.InteractionEventType
 .. currentmodule:: pyvista
 
 .. autodata:: InteractionEventType
-
-
-Concrete Data Object Types
---------------------------
-
```

Comment:
Maybe we keep the docs after all, but rename the TypeVars to `_DataSetType`, etc. to avoid polluting the `pyvista` namespace